assets/css/custom.css

```diff
@@ -229,3 +225,11 @@ color: black; }
     color: rgb(138, 138, 138);
     box-shadow: 0 0 20px rgba(0, 0, 0, 0.541);
 }
+
+.e-mail, .telno-1, .telno-2, .telno-3 {
+    color: white;
+}
+
+.e-mail:hover, .telno-1:hover, .telno-2:hover, .telno-3:hover {
+    color: rgb(221, 221, 221);
+}
```

Comment:
@SilunaP brother you can add a line at the bottom of this file, if not the last line that means } will loss

Reply:
Can you do this also @SilunaP brother